Gemfile.lock

```diff
@@ -1,3 +1,23 @@
+GIT
+  remote: git@github.com:wordpress-mobile/release-toolkit
+  revision: 329effd14e4176817d28ff5b97f35ea84a19c69e
+  branch: trunk
```

Comment:
Pointing the `release-toolkit` to `trunk` in order to get [this fix](https://github.com/wordpress-mobile/release-toolkit/pull/361) for the `429 Too Many Requests` issue we had on GlotPress translations export.